docs/public-networks/reference/cli/options.md

```diff
@@ -408,6 +408,48 @@ When connecting to Mainnet or public testnets, the default is a predefined list
 
 In private networks defined using [`--genesis-file`](#genesis-file) or when using [`--network=dev`](#network), the default is an empty list of bootnodes.
 
+### `cache-last-blocks`
+
+<Tabs>
+
+<TabItem value="Syntax" label="Syntax" default>
+
+```bash
+--cache-last-blocks=<INTEGER>
+```
+
+</TabItem>
+
+<TabItem value="Example" label="Example">
+
+```bash
+--cache-last-blocks=2048
+```
+
+</TabItem>
+
+<TabItem value="Environment variable" label="Environment variable">
+
+```bash
+CACHE_LAST_BLOCKS=2048
+```
+
+</TabItem>
+
+<TabItem value="Example configuration file" label="Example configuration file"> 
+
+```bash
+cache-last-blocks=2048
+```
+
+</TabItem>
+
+</Tabs>
+
+Configures the number of recent blocks to cache, significantly improving the performance of several RPC calls. 
+This caching significantly improves the performance of several RPC calls, particularly [`eth_getBlockByNumber`](../api/index.md#eth_getBlockByNumber), [`eth_getBlockByHash`](../api/index.md#eth_getBlockByHash), [`eth_getTransactionReceipt`](../api/index.md#getTransactionReceipt), and especially [`eth_feeHistory`](../api/index.md#eth_feeHistory). 
```

Comment:
```suggestion
The number of recent blocks to cache. 
Using this option can improve the performance of several RPC calls: [`eth_getBlockByNumber`](../api/index.md#eth_getblockbynumber), [`eth_getBlockByHash`](../api/index.md#eth_getblockbyhash), [`eth_getTransactionReceipt`](../api/index.md#gettransactionreceipt), and especially [`eth_feeHistory`](../api/index.md#eth_feehistory). 
```

Edit for consistency with other option descriptions and to remove wording redundancy.

Also, please always check that your links work properly in the PR preview: https://besu-docs-git-1598-add-flag-hyperledger.vercel.app/development/public-networks/reference/cli/options#cache-last-blocks

Markdown requires heading anchor links to be all lowercase.

Reply:
@alexandratran  Would it be alright to add the word "significantly" to this? The initial PR highlights a substantial improvement in performance, and I believe including this word would more accurately reflect the extent of the enhancement.

Reply:
"Significantly" is a relative term that, IMO, doesn't add any more information to the sentence. I think either specify the "significant improvement" in quantitative terms or leave out the relative term. But not a big deal to use it either so it's up to you.